cpp/velox/operators/plannodes/RowVectorStream.h

```diff
@@ -76,7 +76,7 @@ class ValueStreamNode : public facebook::velox::core::PlanNode {
   }
 
  private:
-  void addDetails(std::stringstream& stream) const {};
+  void addDetails(std::stringstream& stream) const override{};
```

Comment:
format problem, `override {}`

Reply:
I like the style you point, but the style rule checker reject me. see the commit https://github.com/oap-project/gluten/pull/2691/commits/4b7592dcc4bf035426272ee5379d72c196c92385